### DISCUSSION TITLE
How to increase the size of the toolbar in Sumatra?

### DISCUSSION THREAD

**Discussion by emi27777** [sumatrapdfreader/sumatrapdf#4287] [General]
![Captura de pantalla 2024-05-29 100050](https://github.com/sumatrapdfreader/sumatrapdf/assets/170760234/02aeacff-bba5-4c53-b324-38094e989512)

**Comment by kjk** [comment]
Increase size in what sense? Make it physically bigger or add more icons / functionality to toolbar?

To increase toolbar dy use `ToolbarSize = N` (default N is 18) advanced setting.

In [pre-release](https://www.sumatrapdfreader.org/prerelease) you can increase font size of for all UI elements, including toolbar, with `UIFontSize = N` (if 0 we use system value which is 12).

Info about settings: https://www.sumatrapdfreader.org/settings/settings, https://www.sumatrapdfreader.org/docs/Advanced-options-settings

It's not possible to customize toolbar i.e. add more functionality.
